qwik-graphql-tailwind/src/components/issue-pr-card/card.tsx

```diff
@@ -0,0 +1,72 @@
+import { component$ } from '@builder.io/qwik';
+import { format } from 'date-fns';
+import { IssuesIcon, PullRequestIcon } from '../icons';
+import * as styles from './issue-pr-card.classNames';
+
+export interface IssuePrCardProps {
+  data: {
+    url: string;
+    title: string;
+    number: number;
+    isOpen: boolean;
+    createdAt: string;
+    authorName: string;
+    commentsCount: number;
+  };
+  type: 'issue' | 'pr';
+}
+
+export const IssuePrCard = component$(({ data, type }: IssuePrCardProps) => {
+  return (
+    <div className={styles.card_container}>
+      <div className="flex">
+        <label className={styles.card_checkbox}>
+          <input type="checkbox" name="issues[]" autoComplete="off" />
+        </label>
+
+        <div className="flex-shrink-0 pl-4">
+          <span>{type == 'issue' ? <IssuesIcon className="w-5 h-5" /> : <PullRequestIcon className="w-5 h-5" />}</span>
+        </div>
+      </div>
+
+      <div className={styles.card_title_container}>
+        <a className={styles.card_title} href={data.url}>
+          {data.title}
+        </a>
+        <div className={styles.card_subtitle}>
+          <span className="opened-by">
+            #{data.number}
+            {' by '}
+            <a href="#">{data.authorName}</a> was {data.isOpen ? 'opened' : 'closed'} on{' '}
+            {format(new Date(data.createdAt), 'MMM d, yyyy')}
+          </span>
+        </div>
+      </div>
+
+      <div className={styles.card_right_side}>
+        <span className="ml-2 flex-1 flex-shrink-0">
+          {data.commentsCount > 0 && (
+            <a href="#" className="">
+              <div className="flex items-center justify-end">
+                <svg
```

Comment:
can we take this to the icon folder for usability?

Reply:
Sure